.github/workflows/arduino.yml

```diff
@@ -0,0 +1,15 @@
+   name: Arduino CI
+   
+   on: [push]
+    
+   jobs:
+     
+       - name: Fetch sketch dependencies
+          run: |
+       
```

Comment:
Looks better now but [the file is not parsed correctly](https://github.com/DIT112-V20/group-11/actions/runs/327439039). My guess would be there you should remove the empty line from the 9th line.